dopex-io/elvarg
# fix: calcPremiumCustom

## Scope



[closes issue-###](https://link-to-your-issue)

## Implementation

With @pair



## Screenshots

|         | before | after |
| ------- | ------ | ----- |
| desktop |        |       |
| mobile  |        |       |

## How to Test


- check pnl for closed position is accurate
- check amount charged for opening a spread position is close to amount displayed on UI
